tests/test_check_md5.py

```diff
@@ -15,28 +15,36 @@
 
```

Comment:
Could you please help change the file name to `test_check_hash` and add a test for SHA1?
Thanks.

Reply:
this PR includes a sha1 test already, I'll update the name